database/factories/BootcampFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
```

Comment:
Add the line `declare(strict_types=1);` so the code can throw an exception if types do not coincide with the typed requirements.

Reply:
Got it.